src/libraries/utils/ipfs.ts

```diff
@@ -15,8 +16,19 @@ export const uploadToIPFS = async(data: string | Blob | null): Promise<{ hash: s
 	const fetchResult = await fetch(IPFS_UPLOAD_ENDPOINT, {
 		method: 'POST',
 		body: form,
+		headers: {
+			authorization: `Basic ${Buffer.from(`${process.env.INFURA_IPFS_PROJECT_ID}:${process.env.INFURA_IPFS_API_KEY}`).toString('base64')}`,
+		}
 	})
 	const responseBody = await fetchResult.json()
+
+	await fetch(`${IPFS_PIN_ENDPOINT}?arg=${responseBody.Hash}`, {
```

Comment:
afaik, you don't need to pin separately. Just uploading with `pin=true`, should be fine

Reply:
That's correct when using thegraph api. But for Infura, they have two different endpoints for adding/pinning